integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote-heading/cpq-quote-heading.component.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { CpqQuoteHeadingComponent } from './cpq-quote-heading.component';
+import { TranslationService } from '@spartacus/core';
+import { OutletContextData } from '@spartacus/storefront';
+import { BehaviorSubject, of } from 'rxjs';
+describe('Cpq-Quote Heading Component', () => {
```

Comment:
Please, add an empty line between imports and the test description. 
Then rename this test to `CpqQuoteHeadingComponent`